src/emails/verification-email.tsx

```
@@ -1,22 +1,56 @@
+      <Tailwind>
+        <Body className=" font-sans py-[40px]">
+          <Container className="bg-white rounded-[16px] mx-auto p-[48px] max-w-[500px] shadow-sm">
+            <Section className="mb-[32px]">
+              <Text className="text-[24px] font-bold text-[#f59e0b] m-0">10xCoder.club</Text>
+            </Section>
+            <Section>
+              <Heading className="text-[28px] font-bold text-[#111827] m-0 mb-[24px]">
+                Verify your email
+              </Heading>
+              <Text className="text-[16px] leading-[24px] text-[#4b5563] m-0 mb-[16px]">
+                Hey {name},
```

ESLint

[error] `{name}` must be placed on a new line

(style/jsx-one-expression-per-line)